### DISCUSSION TITLE
PlateComponent.cs request

### DISCUSSION THREAD

**Discussion by SweptWasTaken** [space-wizards/space-station-14#3979] [General]
## Description

Requirements
- Can put stuff on/off the plate very easily.
- Plate gives you a percentage buff to whatever you eat.
- If do_after is added to eating, eating with a plate makes you eat much faster.
- When an item is thrown with the plate, it should have a chance to break and spawn whatever it was holding + an entity (99% of the time gonna be shards of plate) <- This thing also needs to account for stuff like plastic plates.

**Comment by mirrorcult** [comment]
dupe #3600 ill close that one since you're the one working on plate-related stuff

**Comment by PaulRitter** [comment]
re: throwing
should prob also account for special behaviour like creampies & not "just" spawn the contents
